NuRadioReco/utilities/signal_processing.py

```diff
@@ -27,3 +31,50 @@ def half_hann_window(length, half_percent=None, hann_window_length=None):
     half_hann_widow[-hann_window_length:] = hann_window[hann_window_length:]
 
     return half_hann_widow
+
+
+def add_cable_delay(station, det, sim_to_data=None, trigger=False, logger=None):
+    """
+    Add or subtract cable delay by modifying the `trace_start_time`.
+
+    Parameters
+    ----------
+    station: Station
+        The station to add the cable delay to.
+
+    det: Detector
+        The detector description
+
+    trigger: bool
+        If True, take the time delay from the trigger channel response.
+        Only possible if `det` is of type `rnog_detector.Detector`. (Default: False)
+
+    logger: logging.Logger, default=None
+        If set, use `logger.debug(..)` to log the cable delay.
```

Comment:
docstring formatting
- use double backticks for single-spaced in rst. Single backticks are interpreted as references
- Can you add a `See Also` / link to the CableDelayAdder class?

Reply:
Like this?